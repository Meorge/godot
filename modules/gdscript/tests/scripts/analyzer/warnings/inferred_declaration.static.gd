func no_exec_test() -> void:
    for i in 10:
        print(i)

func test() -> void:
    pass
