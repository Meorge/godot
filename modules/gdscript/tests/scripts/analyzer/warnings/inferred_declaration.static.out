GDTEST_OK
~~ WARNING at line 2: (INFERRED_DECLARATION) "for" iterator variable "i" has an implicitly inferred static type.
